### PR TITLE
Add script boilerplate for amp js

### LINKED ISSUE #2156: Improve script tag source for amp analytics
**Is your feature request related to a problem? Please describe.**
Raised this issue to address this comment: https://github.com/bbc/simorgh/pull/2105/files#r298190205

We currently define 4 script tags for AMP analytics: 
```
<script key="amp" async src="https://cdn.ampproject.org/v0.js" />
<script
   async
   custom-element="amp-geo"
   src="https://cdn.ampproject.org/v0/amp-geo-0.1.js"
   />
<script
   async
    custom-element="amp-consent"
    src="https://cdn.ampproject.org/v0/amp-consent-0.1.js"
/>
<script
   async
    custom-element="amp-analytics"
    src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"
/>
```

**Describe the solution you'd like**
These scripts are currently used in the following locations:
https://github.com/bbc/simorgh/blob/latest/src/server/Document/component.jsx#L52-L66
https://github.com/bbc/simorgh/blob/latest/.storybook/helpers/ampDecorator/index.jsx

A possible solution is to export the scripts independently in `packages/utilities/psammead-assets/src/amp-boilerplate.js` and then importing where required (see links above).

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**
This may be an invalid issue, as we may require separate scripts, for this issue: https://github.com/bbc/simorgh/issues/1698

### PR DESCRIPTION
**Pull request by samora**
Partially resolves https://github.com/bbc/simorgh/issues/2156

**Overall change:** Add amp JS scripts boilerplate to psammead-assets. See https://github.com/bbc/simorgh/issues/2156

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)

### CODE REVIEW

File: packages/utilities/psammead-assets/src/amp-boilerplate.jsx [hunk 1]
```diff
@@ -1,6 +1,27 @@
 /*
   AMP Boilerplate Code https://github.com/ampproject/amphtml/blob/master/spec/amp-boilerplate.md
 */
+import React from 'react';
 
 export const AMP_SCRIPT = `body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}`;
 export const AMP_NO_SCRIPT = `body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}`;
+export const AMP_JS = (
```
Comment (dr3) [inline packages/utilities/psammead-assets/src/amp-boilerplate.jsx]: Can you split these up please? so theyre 4 separate exports

Comment (samora) [reply packages/utilities/psammead-assets/src/amp-boilerplate.jsx]: Don't you think we should keep them together? They seem to always be used as one.

Comment (dr3) [reply packages/utilities/psammead-assets/src/amp-boilerplate.jsx]: In simorgh yes, but we aim to be reusable :) someone may want to use this for the amp-analytics tag, but not use amp-consent etc :)

**Review by dr3 — COMMENTED** [review]
Looks good :) Just needs some snapshot tests added :)

**Review by hotinglok — COMMENTED** [review]
Jenkins is yelling about stuff, but overall LGTM.

**Review by jroebu14 — APPROVED** [review]

**Review by rhenshaw56 — APPROVED** [review]

**Comment by amywalkerdev** [comment]
Tests are failing in packages/utilities/psammead-assets/index.test.jsx

Psammead assets › should test all the utility exports exist and are the correct type for main

    Missing value 'AMP_JS' in the expected export for 'psammead-assets/amp-boilerplate'.

Psammead assets › should test all the utility exports exist and are the correct type for module

    Missing value 'AMP_JS' in the expected export for 'psammead-assets/amp-boilerplate'.

**Comment by amywalkerdev** [comment]
Change scanner is complaining:

Branch must update CHANGELOG.md in psammead
Branch must update package.json in psammead

I would recommend removing your changes to the base package-lock.json as they are not changing anything useful :)

**Review by amywalkerdev — APPROVED** [review]

**Pull request merged**